api/model/src/main/java/org/projectnessie/model/Conflict.java

```diff
@@ -34,10 +34,11 @@
     ignoreUnknown = true)
 public interface Conflict {
   @Value.Parameter(order = 1)
-  @Nullable
-  @jakarta.annotation.Nullable
   @JsonDeserialize(using = ConflictType.Deserializer.class)
-  ConflictType conflictType();
+  @Value.Default
+  default ConflictType conflictType() {
+    return ConflictType.UNKNOWN;
```

Comment:
This helps with the case where the field is not even present in the JSON payload. Not sure how realistic that is though.

Reply:
Since we're removing `null` handling from `deserialize()`, I guess we do not have to provide a default here. All server-generated values will have a type, as you noted previously.

Reply:
I was about to suggest the same thing :-)